debugoverlay-ext-netstats/src/main/java/com/ms_square/debugoverlay_ext_netstats/NetStatsDataModule.java

```diff
@@ -0,0 +1,76 @@
+package com.ms_square.debugoverlay_ext_netstats;
+
+import android.net.TrafficStats;
+import android.os.Handler;
+import android.os.Looper;
+
+import com.ms_square.debugoverlay.modules.BaseDataModule;
+
+import java.util.Locale;
+
+public class NetStatsDataModule extends BaseDataModule<String> {
+
+    private static final String TAG = NetStatsDataModule.class.getSimpleName();
+
+    private static final String HEADER = "Received: %8s/s\nSent: %12s/s";
+
+    private Handler handler = new Handler(Looper.getMainLooper());
+
+    private final int interval;
+    private final int uid;
+
+    private double mPreviousReceived = 0;
+    private double mPreviousSent = 0;
+    private double mReceived;
+    private double mSent;
+
+    public NetStatsDataModule(int interval, int uid) {
+        this.interval = interval;
+        this.uid = uid;
+    }
+
+    @Override
+    protected String getLatestData() {
+        return String.format(
+                Locale.ENGLISH, HEADER, bytesToPrettyString(mReceived), bytesToPrettyString(mSent));
```

Comment:
I don't think it would make any difference in this case, but could u use Locale.US just to be consistent?